### PR TITLE
#629 Thumbnails of crops on post

### PR DESCRIPTION
**Pull request by oshiho3**
This is an experiment for https://github.com/Growstuff/growstuff/issues/629.

The thumbnails placed at the bottom.

### CODE REVIEW

File: app/views/posts/show.html.haml [hunk 1]
```diff
@@ -16,16 +16,24 @@
...
-      = link_to 'Edit Post', edit_post_path(@post), :class => 'btn btn-default btn-xs'
-    - if can? :destroy, @post
-      = link_to 'Delete Post', @post, method: :delete, |
-        data: { confirm: 'Are you sure?' }, :class => 'btn btn-default btn-xs'
+- unless @post.crops.empty?
+  %h3 Crops mentioned in this post
+  - @post.crops.each do |c|
+    - unless c.photos.empty?
+      .col-md-2
+        = render :partial => 'crops/thumbnail', :locals => { :crop => c }
```
Comment (tygriffin) [inline app/views/posts/show.html.haml]: I think we should show a thumbnail even if there are no photos. The way it is now, potentially the `h3` can be shown with no thumbnails underneath it.

Comment (oshiho3) [inline app/views/posts/show.html.haml]: Thanks for checking. absolutely. done

**Comment by Skud** [comment]
Can we see a thumbnail of how this looks, please?

**Comment by oshiho3** [comment]
The screenshot was uploaded in #629. Please see the latest one.

**Comment by maco** [comment]
@oshiho3 looks like there are two different PRs on this branch?

**Comment by oshiho3** [comment]
@maco, does it? I'm not sure what that means. I meant just one PR. The title of the PR originally had the term "WIP", but I removed it from the title it after.

**Comment by maco** [comment]
The diff on this includes changes for "only show approved crops"

**Comment by oshiho3** [comment]
I believe the latest PR, that filters out unapproved crops, should be the one to be merged.

**Comment by cesy** [comment]
Does this PR need updating, merging or closing, then, @oshiho3 ?

**Comment by oshiho3** [comment]
This should be good to be merged.

**Comment by cesy** [comment]
Reviewed and this looks good to me.

**Pull request merged**